### PR TITLE
Trac 58833 -  Docs: improve comments, to align with Coding Standards.

### PR DESCRIPTION
**Pull request by kebbet**
Fixing minor style issues in the comments, after `6.3.2`.

changed to https://core.trac.wordpress.org/ticket/59651

### CODE REVIEW

**Comment by kebbet** [comment]
Please review @mukeshpanchal27 or @costdev :)

File: src/wp-includes/rest-api/class-wp-rest-server.php [hunk 1]
```diff
@@ -486,13 +486,13 @@ public function serve_request( $path = null ) {
...
-		 * @since 6.3.2 Moved the block to catch the filter added on rest_cookie_check_errors() from rest-api.php
+		 * @since 6.3.2 Moved the block to catch the filter added on rest_cookie_check_errors() from rest-api.php.
 		 *
 		 * @param bool $rest_send_nocache_headers Whether to send no-cache headers.
 		 */
 		$send_no_cache_headers = apply_filters( 'rest_send_nocache_headers', is_user_logged_in() );
 
-		// send no cache headers if the $send_no_cache_headers is true
+		// Send no cache headers if the $send_no_cache_headers is true
 		// OR if the HTTP_X_HTTP_METHOD_OVERRIDE is used but resulted a 4x response code.
```
Comment (costdev) [inline src/wp-includes/rest-api/class-wp-rest-server.php]:
```suggestion
		/*
		 * Send no cache headers if the $send_no_cache_headers is true,
		 * OR if the HTTP_X_HTTP_METHOD_OVERRIDE is used but resulted a 4x response code.
		 */
```
Let's fix these lines to use the multiline comment format per [the inline documentation standards](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#5-2-multi-line-comments) while we're here. 🙂 

Also added a comma to the end of the first line as I initially thought this needed a period, until I saw it was (incorrectly) using the single-line comment format.

**Comment by kebbet** [comment]
Agree on that, thank you for your comment and review Colin!

File: src/wp-includes/rest-api/class-wp-rest-server.php [hunk 2]
```diff
@@ -486,14 +486,16 @@ public function serve_request( $path = null ) {
...
 		 *
 		 * @param bool $rest_send_nocache_headers Whether to send no-cache headers.
 		 */
 		$send_no_cache_headers = apply_filters( 'rest_send_nocache_headers', is_user_logged_in() );
 
-		// send no cache headers if the $send_no_cache_headers is true
-		// OR if the HTTP_X_HTTP_METHOD_OVERRIDE is used but resulted a 4x response code.
+		/*
+		 * Send no cache headers if the $send_no_cache_headers is true
+		 * OR if the HTTP_X_HTTP_METHOD_OVERRIDE is used but resulted a 4x response code.
```
Comment (mukeshpanchal27) [inline src/wp-includes/rest-api/class-wp-rest-server.php]:
```suggestion
		 * Send no cache headers if the $send_no_cache_headers is true,
		 * OR if the HTTP_X_HTTP_METHOD_OVERRIDE is used but resulted a 4x response code.
```

Add missing comma

Comment (kebbet) [reply src/wp-includes/rest-api/class-wp-rest-server.php]: Thanks, fixed.

**Review by mukeshpanchal27 — APPROVED** [review]
Thanks @kebbet for the changes. LGTM!

**Comment by kebbet** [comment]
Please review and feel free to approve (and commit) if you have the time to do it @costdev !

**Comment by SergeyBiryukov** [comment]
Thanks for the PR! Merged in [r57120](https://core.trac.wordpress.org/changeset/57120).

**Pull request closed without merging**